src/mono/mono.proj

```diff
@@ -73,6 +76,199 @@
+      <_MonoConfigureParams Include="--disable-hybrid-suspend" />
+      <_MonoConfigureParams Include="--disable-crash-reporting" />
+
+      <_MonoAC_VARS Include="ac_cv_c_bigendian=no" />
+      <_MonoAC_VARS Include="ac_cv_func_fstatat=no" />
+      <_MonoAC_VARS Include="ac_cv_func_readlinkat=no" />
+      <_MonoAC_VARS Include="ac_cv_func_getpwuid_r=no" />
+      <_MonoAC_VARS Include="ac_cv_func_posix_getpwuid_r=yes" />
+      <_MonoAC_VARS Include="ac_cv_header_curses_h=no" />
+      <_MonoAC_VARS Include="ac_cv_header_localcharset_h=no" />
+      <_MonoAC_VARS Include="ac_cv_header_sys_user_h=no" />
+      <_MonoAC_VARS Include="ac_cv_func_getentropy=no" />
+      <_MonoAC_VARS Include="ac_cv_func_futimens=no" />
+      <_MonoAC_VARS Include="ac_cv_func_utimensat=no" />
+      <_MonoAC_VARS Include="ac_cv_func_shm_open_working_with_mmap=no" />
+      <_MonoAC_VARS Include="mono_cv_sizeof_sunpath=104" />
+      <_MonoAC_VARS Include="mono_cv_uscore=yes" />
+      <_MonoAC_VARS Include="ac_cv_func_system=no" />
+      <_MonoAC_VARS Include="ac_cv_func_pthread_kill=no" />
+      <_MonoAC_VARS Include="ac_cv_func_kill=no" />
+      <_MonoAC_VARS Include="ac_cv_func_sigaction=no" />
+      <_MonoAC_VARS Include="ac_cv_func_fork=no" />
+      <_MonoAC_VARS Include="ac_cv_func_execv=no" />
+      <_MonoAC_VARS Include="ac_cv_func_execve=no" />
+      <_MonoAC_VARS Include="ac_cv_func_execvp=no" />
+      <_MonoAC_VARS Include="ac_cv_func_signal=no" />
+
+      <_MonoCFLAGS Condition="'$(Platform)' == 'arm64'" Include="-arch arm64" />
+      <_MonoCFLAGS Include="-isysroot $(XcodeDir)/Platforms/AppleTVOS.platform/Developer/SDKs/AppleTVOS$(tvOSVersion).sdk" />
+      <_MonoCFLAGS Include="-mtvos-version-min=$(tvOSVersionMin)" />
+      <_MonoCFLAGS Include="-Wl,-application_extension" />
+      <_MonoCFLAGS Include="-fexceptions" />
+
+      <_MonoCXXFLAGS Condition="'$(Platform)' == 'arm64'" Include="-arch arm64" />
+      <_MonoCXXFLAGS Include="-isysroot $(XcodeDir)/Platforms/AppleTVOS.platform/Developer/SDKs/AppleTVOS$(tvOSVersion).sdk" />
+      <_MonoCXXFLAGS Include="-mtvos-version-min=$(tvOSVersionMin)" />
+      <_MonoCXXFLAGS Include="-Wl,-application_extension" />
+
+      <_MonoCPPFLAGS Condition="'$(Platform)' == 'arm64'" Include="-arch arm64" />
+      <_MonoCPPFLAGS Include="-isysroot $(XcodeDir)/Platforms/AppleTVOS.platform/Developer/SDKs/AppleTVOS$(tvOSVersion).sdk" />
+      <_MonoCPPFLAGS Include="-mtvos-version-min=$(tvOSVersionMin)" />
+      <_MonoCPPFLAGS Include="-DMONOTOUCH=1" />
+      <_MonoCPPFLAGS Include="-DSMALL_CONFIG" />
+      <_MonoCPPFLAGS Include="-D_XOPEN_SOURCE" />
+      <_MonoCPPFLAGS Include="-DHOST_IOS" />
+      <_MonoCPPFLAGS Include="-DHOST_TVOS" />
+      <_MonoCPPFLAGS Include="-DHAVE_LARGE_FILE_SUPPORT=1" />
+
+      <_MonoLDFLAGS Condition="'$(Platform)' == 'arm64'" Include="-arch arm64" />
+      <_MonoLDFLAGS Include="-Wl,-no_weak_imports" />
```

Comment:
missing a flag:
```suggestion
      <_MonoLDFLAGS Include="-Wl,-no_weak_imports" />
      <_MonoLDFLAGS Include="-Wl,-bitcode_bundle" />
```

Reply:
This causes the build to fail with `-undefined and -bitcode_bundle cannot be used together`